docs/en/how_to_guides/finetune_with_a_custom_dataset.md

```diff
@@ -0,0 +1,357 @@
+
+MindCV uses [` create_optimizer `](https://mindspore-lab.github.io/mindcv/zh/reference/optim/#mindcv.optim.optim_factory.cre ate_optimizer) to generate the optimizer and passes the learning rate to the optimizer. To set the tiered learning rate, simply **change the `params` parameter of `create_optimizer` function in finetune.py from `network.trainable_params()` to a list containing the names of the specific parameters and the corresponding learning rate**, which you can refer to the [API documentation of optimizers](https://www.mindspore.cn/docs/zh-CN/r2.0/api_python/mindspore.nn.html#%E4%BC%98%E5%8C%96%E5%99%A8). The specific structure of the network and the parameter names in each layer can be viewed by printing the result of `create_model` -- `network`.
+
+> Tips: You can also use the same operation to set different weight_decay for parameters.
+
+#### Set Learning Rate for Classifier
+
+Taking mobilenet v3-small as an example, the model classifier name starts with "classifier", so if we only increase the learning rate of the classifier, we need to specify it at each step of training. `lr_scheduler` is a learning rate list generated by `create_scheduler`, which contains the learning rate at each step of training. Suppose we adjust the learning rate of the classifier to 1.2 times that on the feature network. The changes to the finetune.py code are as follows:
+
+```python
+# ...
+
+
+# Note: a)the params-lr dict must contain all the parameters. b)Also, you're recommended to set a dict with a key "order_params" to make sure the parameters will be updated in the right order.
+params_lr_group = [{"params": list(filter(lambda x: 'classifier' in x.name, network.trainable_params())),
+                    "lr": [i*1.2 for i in lr_scheduler]},
+                   {"params": list(filter(lambda x: 'classifier' not in x.name, network.trainable_params())),
+                    "lr": lr_scheduler}，
+                  {"order_params": network.trainable_params()}]
+
+optimizer = create_optimizer(params_lr_group,
+                             opt=args.opt,
+                             lr=lr_scheduler,
+                             ...)
+```
+
+#### Set Learning Rate for Specific Layers
```

Comment:
Duplicate with line 259. 

We may combine these sections for param grouping into one, e.g.
```
### Param Grouping: Set Different Learning Rates for Feature Layers and Classifier Layers 
```

Reply:
Just make the subtitile more specific. `for classifier(267)` or `for specific layers(278)` are two excellent examples. Replace `for specific layers(278)` with `for whatever layers you want`? Still, it is an exciting example.